tests/data_context/store/test_store.py

```diff
@@ -69,3 +71,65 @@ def test_build_store_from_config_failure(store_config: dict, module_name: str):
             store_config=store_config,
             module_name=module_name,
         )
+
+
+@pytest.mark.unit
+def test_store_add_success():
+    store = Store()
+    key = StringKey("foo")
+    value = "bar"
+
+    store.add(key=key, value=value)
+    assert store.has_key(key)
```

Comment:
`non-blocking`/out-of-scope
I see this a lot... 
Why do we use `.has_key()`? 
The canonical python way to do this is with the `in` / `__contains__` operator.
https://docs.python.org/3/library/operator.html#operator.__contains__

Seems like our `stores` should implement/inherit the `Collections.ABC`.
https://docs.python.org/3/library/collections.abc.html#collections-abstract-base-classes

Reply:
ugh yeah I'm not sure tbh - this is a super legacy API. Don and I want to change things but not sure if this is the time to do so